.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
In the headings, you switch between "normal" case and title case (e.g. "Describe the bug", but then "To Reproduce") - please be consistent. I prefer title case. This also applies to the other templates.

Reply:
-> 40aad032b30af2845984b1dcb47a52684ac2434b